.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "mocha": true
   },
   "globals": {
-    "document": false
+    "document": "readonly"
```

Comment:
Sorry, stupid question. Why do we want to disallow writing to the document? Do we have a library/framework which is preferred over writing directly to the DOM? Does Electron supply something like that? It's also likely I've misunderstood this rule and/or Electron.

Reply:
The DOM can still be modified (`document.appendChild(el)` is ok) but the *variable* will not be allowed to be overwritten (`document = {}` is not ok)

https://eslint.org/docs/user-guide/configuring#specifying-globals